app/Console/Command/UpdatePlaceLocality.php

```diff
@@ -76,33 +98,42 @@ public function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): ?int
     {
+        $itemType = new ItemType($input->getArgument(self::ITEM_TYPE));
+        if ($itemType->sameAs($itemType::event())) {
+            $output->writeln('<error>Only itemType place or organizer are accepted.</error>');
+            return self::FAILURE;
+        }
         $query = $input->getArgument(self::QUERY);
         $newLocality = $input->getArgument(self::NEW_LOCALITY);
 
         if ($query === null || $newLocality === null) {
-            $output->writeln('<error>Missing argument, the correct syntax is: place:update-locality "sapi3_query" "new_locality"</error>');
+            $output->writeln('<error>Missing argument, the correct syntax is: place:update-locality "item_type" "sapi3_query" "new_locality"</error>');
```

Comment:
I don't think the names in the error method match with the actual values of the constants?